client-core/src/main/java/com/influxdb/query/FluxRecord.java

```diff
@@ -121,6 +124,14 @@ public Map<String, Object> getValues() {
         return values;
     }
 
+    /**
+     * @return record's columns
+     */
+    @Nonnull
+    public ArrayList<Object> getRow() {
```

Comment:
Please use `generic` interface:

```suggestion
    public List<Object> getRow() {
```